src/AppInstallerCommonCore/MSStoreDownload.cpp

```diff
@@ -910,6 +911,13 @@ namespace AppInstaller::MSStore
             {
                 SFS::RequestParams sfsClientRequest;
                 sfsClientRequest.productRequests = { {std::string{ wuCategoryId }, {}} };
+                if (AppInstaller::Settings::Network().GetProxyUri())
+                {
+                    std::string proxyUri = AppInstaller::Settings::Network().GetProxyUri().value();

```

Comment:
I just realized that `GetProxyUri()` returns a copy instead of a reference. We should add `&` in `NetworkSettings.h` to avoid these two lines from making an unnecessary copy

Reply:
Fixed, thanks Flor